cds/modules/deposit/api.py

```diff
@@ -787,6 +793,39 @@ def _tasks_global_status(self):
         iterate_events_results(events=events, fun=global_status)
         return global_status.status
 
+    def _rename_subtitles(self):
+        pattern = re.compile('(?P<filename>.*)_(?P<rest>[a-zA-Z]{2}\.vtt$)')
```

Comment:
Could you, please, put a comment what this regexp is exactly doing? It will be easier to debug/change it in the future.

Reply:
I would also propose `rest` -> `iso_lang` for example

Reply:
ok but this regex is the same as the one in the previewer module. Here https://github.com/CERNDocumentServer/cds-videos/blob/38891af5d9e2ee995e76490740148c43873c5b10/cds/modules/previewer/api.py#L143